services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -137,7 +137,7 @@ class ApplicationSettings(BasicSettings):
     API_SERVER_DIRECTOR_V2: DirectorV2Settings | None = Field(
         auto_default_from_env=True
     )
-    API_SERVER_MAX_LOG_CHECK_SECONDS: NonNegativeInt = 30
+    API_SERVER_LOG_CHECK_TIMEOUT_SECONDS: NonNegativeInt = 3 * 60
```

Comment:
NOTE: I noticed this var is not exposed in the `docker-compose.services[api-server].environments` section. That means that the auto-deployer does not know anything about it. As a consequence it will not be added in portainer environments and if you add it by hand (e.g. to check new times) it will be removed as soon as the auto-deployer runs.

Reply:
Yes, I am aware of this. I don't see any reason to want to tune it right now. So I wouldn't bother about setting it in the docker compose. Let me know if you disagree.